app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -30,9 +24,8 @@ const DeleteAlert = ({
       isOpen
       isSubmitting={deleting}
       message="Are you sure you want to continue? This cannot be undone."
```

Comment:
Go through the Slack thread and fix the issue over here.

Reply:
I have pushed commits to resolve the issues from the last PR and this one along with this PR